### PR TITLE
test(highlight): enhance test coverage

### LINKED ISSUE #785: Enhance Test Coverage for `@yamada-ui/highlight`
related: #775

### Description
Enhance test coverage for components with less than 80% coverage to meet or exceed 80%.

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### PR DESCRIPTION
**Pull request by txxxxc**
Closes #785 

## Description
Enhance test coverage for components with less than 80% coverage to meet or exceed 80%.

## Is this a breaking change (Yes/No):
No

## Additional Information
This is my first pull request for OSS, so if you have any suggestions for improvement, I would appreciate it.

### CODE REVIEW

**Review by taroj1205 — CHANGES_REQUESTED** [review]
Hi txxxxc,
Thank you for your PR.
Please run
```
pnpm run format
```
in your terminal to format your code with prettier.

File: packages/components/highlight/tests/highlight.test.tsx [hunk 1]
```diff
@@ -20,6 +20,37 @@ describe("<Highlight/>", () => {
...
+      <Highlight isFragment query="">
+        Highlight
+      </Highlight>
+    )
+    expect(container.firstChild?.nodeName).toBe("#text")
+  })
+
+  test("markProps prop works correctly", () => {
+    const { getByText } = render(
+      <Highlight query="Highlight" markProps={{ borderRadius: '12px' }}>
```
[the remark this answers is not in the document]
Comment (txxxxc) [reply packages/components/highlight/tests/highlight.test.tsx]: resolved in https://github.com/yamada-ui/yamada-ui/pull/925/commits/8230096307394b9b67ce73e539cef823c364362d

**Comment by txxxxc** [comment]
Thank you for reviewing. Could you please check the PR again?

**Review by taroj1205 — APPROVED** [review]
Approved.

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**